packages/commerce-sdk-react/src/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
I was talking with chat gpt.. and I was wondering if the type that we are looking at is simply the type of the console object. Like this:

```
type Logger = typeof console
```